static/app/components/autoComplete.tsx

```diff
@@ -115,12 +115,11 @@ type Props<T> = typeof defaultProps & {
   disabled: boolean;
   defaultHighlightedIndex?: number;
   defaultInputValue?: string;
-  /**
-   * Currently, this does not act as a "controlled" prop, only for initial state of dropdown
-   */
+  inputValue?: string;
   isOpen?: boolean;
   itemToString?: (item?: T) => string;
   onClose?: (...args: Array<any>) => void;
+  onInputValueChange?: (value: string) => void;
```

Comment:
I could have used `inputProps` to pass the onChange handler but I thought that mirroring the Downshift API would be more intuitive